### ISSUE TITLE
Feature request: please accept_empty_strings

### ISSUE THREAD

**Issue by joaomcferreira** [casper2020/pg-jsonapi#7]
Presently, the "request-accounting-schema-function-arg" and "request-accounting-prefix-function-arg" options can be used to ask jsonapi to map those 2 values to their respective SQL function arguments. That is great and useful and I love it. Perfect :)

But... in that scenario jsonapi will reject the request if any of those 2 strings is empty (like ''). This renders that functionality unusable (and that's a big loss in my opinion) for the cases where those string are naturally empty. That is the case for situations where the user did not select any fiscal year or maybe he does not have the CTB module active.

So: it would be great if an SQL function programmer could simply ask jsonapi to "please, accept_empty_strings, I now what I'm doing".

Put simply: "accept_empty_strings"

This would be just great

**Comment by andreCatita** [comment]
I'm unable to check or test right now, but I think you can just default null in the target function?
